vms/proposervm/block.go

```diff
@@ -193,13 +203,24 @@ func (p *postForkCommonComponents) buildChild(
 		return nil, err
 	}
 
-	shouldBuildUnsignedBlock, err := p.shouldBuildUnsignedBlock(
-		ctx,
-		parentID,
-		parentTimestamp,
-		parentPChainHeight,
-		newTimestamp,
-	)
+	shouldBuildUnsignedBlock := false
```

Comment:
Should we add a comment here like `shouldHaveProposer` that explains this is the default in Durango?